quickwit/quickwit-serve/src/cluster_api/rest_handler.rs

```diff
@@ -29,7 +29,7 @@ use crate::Format;
 /// Cluster handler.
 pub fn cluster_handler(
     cluster: Arc<Cluster>,
-) -> impl Filter<Extract = impl warp::Reply, Error = Rejection> + Clone {
+) -> impl Filter<Extract = (impl warp::Reply,), Error = Rejection> + Clone {
```

Comment:
I had some weird warnings, I need to figure it out. You can ignore all the changes of `Filter`. Sorry for the noise

```
warning: opaque type `impl warp::Filter + std::clone::Clone + warp::filter::FilterBase<Extract = impl warp::Reply, Error = warp::Rejection>` does not satisfy its associated type bounds
  --> quickwit-serve/src/cluster_api/rest_handler.rs:32:18
   |
32 | ) -> impl Filter<Extract = impl warp::Reply, Error = Rejection> + Clone {
   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
  ::: /Users/fmassot/.cargo/registry/src/github.com-1ecc6299db9ec823/warp-0.3.3/src/filter/mod.rs:40:19
   |
40 |     type Extract: Tuple; // + Send;
   |                   ----- this associated type bound is unsatisfied for `impl warp::Reply`
   |
   = note: `#[warn(opaque_hidden_inferred_bound)]` on by default
help: add this bound
   |
32 | ) -> impl Filter<Extract = impl warp::Reply + warp::generic::Tuple, Error = Rejection> + Clone {
```